### PR TITLE
Support for showing and de-selecting filters

### PR DESCRIPTION
**Pull request by cnunciato**
Accompanies https://github.com/pulumi/registry/pull/82.

### CODE REVIEW

File: src/scss/_packages.scss [hunk 1]
```diff
@@ -8,116 +8,137 @@
...
+            [data-selected-types*="component"] & {
+                @apply bg-purple-300;
             }
         }
+    }
 
-        [data-category="cloud"] {
-            .package-tag-category {
-                @apply border-green-300;
+    [data-filter-group="type"] {
```
Comment (cnunciato) [inline src/scss/_packages.scss:67]: Ooh, this is a straight-up dupe! I'll fix.

File: src/ts/packages.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 $(".section-registry").on("filterSelect", (event) => {
```
Comment (cnunciato) [inline src/ts/packages.ts:1]: This jQuery event-handling stuff was just me trying to be pragmatic; time-permitting I'd like to pull it up into a wrapper component, now that things are beginning to settle.

**Review by cnunciato — COMMENTED** [review]
Will add some comments to these files as well.

**Review by kimberleyamackenzie — APPROVED** [review]

**Review by djgrove — APPROVED** [review]
👍

**Pull request merged**